content/en/cloud_cost_management/real_time_costs.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Real-Time Costs
+private: true
+description: View and analyze cloud spend in real time.
+further_reading:
+- link: "/cloud_cost_management/"
+  tag: "Documentation"
+  text: "Learn about Cloud Cost Management"
+---
+
+{{< callout url="https://www.datadoghq.com/product-preview/real-time-costs" header="Join the Preview!">}}
+Real-Time Costs is in Preview.
+{{< /callout >}}
+
+## Overview
+
+Real-Time Costs provide near real-time estimates of your Amazon EC2 costs, including Kubernetes cost allocation, so that you can react to cost changes in minutes and hours, not days. Estimates are generated using real-time usage data from the Datadog Agent, based on recent average hourly net amortized EC2 prices by instance type, region, and AWS account.
```

Comment:
```suggestion
Real-Time Costs provide near real-time estimates of your Amazon EC2 costs, including Kubernetes cost allocation, so that you can react to cost changes in minutes and hours, instead of days. Estimates are generated using real-time usage data from the Datadog Agent, based on recent average hourly net amortized EC2 prices by instance type, region, and AWS account.
```